src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -118,7 +118,7 @@
 $COMPOSER_JSON = getenv('COMPOSER') ?: 'composer.json';
 
 $root = __DIR__;
-while (!file_exists($root.'/'.$COMPOSER_JSON) || file_exists($root.'/DeprecationErrorHandler.php')) {
+while (!file_exists($root.'/'.$COMPOSER_JSON) || !file_exists($root.'/DeprecationErrorHandler.php')) {
```

Comment:
I don't understand this code, why the whole loop can't be just replaced with:

```php
$root = dirname(__DIR__);
```;

the root needs to be a directory of `symfony/phpunit-bridge`, so the `DeprecationErrorHandler.php` is in that directory, right?

Reply:
The DeprecationErrorHandler check was added in:
- https://github.com/symfony/phpunit-bridge/commit/a558e6a6a754297c8aacd2f179cec54fe343ca4d

Reply:
Yeah, I can see what the code does, but why is all that needed? what's wrong with my proposal:
- https://github.com/symfony/symfony/pull/45930#discussion_r841259869

what's the dynamic part needed? `vendor/symfony/phpunit-bridge` structure is always consistent, i.e what is in git repo.